src/ai/backend/manager/idle.py

```diff
@@ -307,24 +305,133 @@ class UtilizationIdleChecker(BaseIdleChecker):
     """
 
     name: ClassVar[str] = "utilization"
-    default_cpu_util_threshold: ClassVar[float] = 30.0
-    default_accelerator_util_threshold: ClassVar[float] = 10.0
-    default_initial_grace_period: ClassVar[float] = 300.0  # allow first 5 minutes to be idle
+    cpu_util_series: List[float] = []
+    mem_util_series: List[float] = []
 
-    async def populate_config(self, config: Mapping[str, Any]) -> None:
-        pass
+    idle_timeout: timedelta
 
-    async def check_session(self, session: Row, dbconn: SAConnection) -> bool:
-        # last_stat = session['last_stat']
-        # TODO: implement
-        return True
+    _config_iv = t.Dict(
+        {
+            t.Key("threshold", default="10m"): tx.TimeDuration(),
+        }
+    ).allow_extra("*")
+
+    _evhandlers: List[EventHandler[None, AbstractEvent]]
+
+    async def populate_config(self, raw_config: Mapping[str, Any]) -> None:
+        config = self._config_iv.check(raw_config)
+        self.idle_timeout = config["threshold"]
```

Comment:
Do we use `checkers/timeout/threshold` for the utilization check period? If this is needed, wouldn't it be better to separate out as `checkers/utilization/check-period` or something like that? In case there is an admin who wants to enable utilization checker, but not idle timeout, still we need to write both of the configs in the current implementation.

Reply:
Also, the raw_config here does not carry the `threshold` item since `raw_config` should be a config mapping under `/config/idle/utilization`.